src/components/ContinuationIn/SummaryBusinessHomeJurisdiction.vue

```diff
@@ -273,28 +287,31 @@ export default class SummaryDefineCompany extends Vue {
     return null
   }
 
-  get identifyingNumber (): string {
-    return this.getContinuationInBusinessInfo?.homeIdentifier
+  /** The identifier in the home jurisdiction. */
+  get identifier (): string {
+    return this.getExistingBusinessInfo?.identifier
```

Comment:
The reason I'm saying "in the home jurisdiction" (or, in other places, "in BC") is because we may have both the identifier and legal name in both. I'm trying to be explicit in the code to reduce confusion.